modules/test/playwright/pages/users-admin-web/UsersAndOrganizationsPage.ts

```diff
@@ -469,6 +472,12 @@ export class UsersAndOrganizationsPage {
 		this.usersTable = page.locator(
 			'#_com_liferay_users_admin_web_portlet_UsersAdminPortlet_usersSearchContainer'
 		);
+		this.usersTableCell = (userName: string) => {
+			return this.page.getByRole('cell', {
+				exact: true,
+				name: `${userName}`,
```

Comment:
nice to have: as you're not "building" a string but it's just the value of the variable you can just do
```
name: userName
```
😄